tests/validators/invalid_urls.txt

```diff
@@ -50,3 +50,4 @@ http://[::1:2::3]:8080/
 http://[]
 http://[]:8080
 http://example..com/
+://example.com
```

Comment:
It was supposed to fail since by default 'http(s)', 'ftp(s)' are expected. But `URLValidator(schemes=[''])('://www.google.com')` was passing on master which will not pass now.